arch/generic/Makefile.in

```diff
@@ -57,6 +58,13 @@ insert_string_c.o: $(SRCDIR)/insert_string_c.c  $(SRCTOP)/zbuild.h $(SRCTOP)/def
 insert_string_c.lo: $(SRCDIR)/insert_string_c.c  $(SRCTOP)/zbuild.h $(SRCTOP)/deflate.h $(SRCTOP)/insert_string_tpl.h
 	$(CC) $(SFLAGS) $(INCLUDES) -c -o $@ $(SRCDIR)/insert_string_c.c
 
+slide_hash_c.o: $(SRCDIR)/slide_hash_c.c  $(SRCTOP)/zbuild.h $(SRCTOP)/deflate.h
+	$(CC) $(CFLAGS) $(INCLUDES) -c -o $@ $(SRCDIR)/slide_hash_c.c
+
+slide_hash_c.lo: $(SRCDIR)/slide_hash_c.c  $(SRCTOP)/zbuild.h $(SRCTOP)/deflate.h
+	$(CC) $(SFLAGS) $(INCLUDES) -c -o $@ $(SRCDIR)/slide_hash_c.c
+
+
```

Comment:
Is two new lines here intentional?

Reply:
It was not intentionally added in that commit, but I had intended to keep 2 empty lines above and below that list from the beginning, to provide a bit more separation in the file. IMO it looks cleaner, but that is just my personal preference.

Reply:
Ok.